crates/solidity/outputs/npm/tests/src/tests/ast.ts

```diff
@@ -1,145 +0,0 @@
-import { Language } from "@nomicfoundation/slang/language";
```

Comment:
Same suggestion for these tests, copying/forking the tests rather than moving.

I'm a bit uncomfortable about removing such E2E tests for Solidity and its types. This is the only coverage it has.

Reply:
As per my other comment on this topic